src/pages/en/guides/deploy/flightcontrol.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Deploy your Astro Site to AWS with Flightcontrol
+description: How to deploy your Astro site to AWS with Flightcontrol
+layout: ~/layouts/DeployGuideLayout.astro
+---
+
+[Flightcontrol](https://www.flightcontrol.dev?ref=astro) enables any developer to deploy to AWS without being a wizard. It's extremely easy to use but lets you pop the hood and leverage the raw power of AWS when needed.
+
+Supports both static and SSR Astro sites.
```

Comment:
```suggestion
You can deploy an Astro site using [Flightcontrol](https://www.flightcontrol.dev?ref=astro), which provides fully-automated deployments to AWS.
```
This is more in keeping with the neutral tone of the other pages. We typical start with "You can deploy an Astro site using..." and then a very factual (not marketing) description. So, I'd suggest something like the above, but do correct if it's factually inaccurate.